### PR TITLE
Feat: Necessary UI Changes

### PR DESCRIPTION
**Pull request by hussiantabha**
Visible changes on the home page and got rid of the cart error

### CODE REVIEW

File: src/Components/Main.js [hunk 1]
```diff
@@ -43,3 +52,7 @@ const Main = () => {
   );
 };
 export { Main };
+
+{
+  /* <p>No matter where you are in the world, you are at home when tea is served</p>; */
```
Comment (t007rushi) [inline src/Components/Main.js]: rmv comment

File: src/Components/listingProducts.js [hunk 2]
```diff
@@ -54,7 +52,7 @@ const ListingProducts = () => {
       console.log(convertedJSON);
```
Comment (t007rushi) [inline src/Components/listingProducts.js]: console to remove

File: src/Pages/Cart.js [hunk 3]
```diff
@@ -97,86 +111,94 @@ const Cart = () => {
...
+                      </div>
+                      <div className="cart-card-content">
+                        <h2>{product.productBrand}</h2>
+                        <div className="card-price">
+                          <span className="price-scratch">
+                            ₹{product.originalPrice}
+                          </span>
+                          <span>₹{product.price}</span>
+                        </div>
+                        {/* <span>45% Off</span> */}
```
Comment (t007rushi) [inline src/Pages/Cart.js]: comment if not required

File: src/Pages/Cart.js [hunk 4]
```diff
@@ -97,86 +111,94 @@ const Cart = () => {
...
+      {userLoggedIn ? (
+        cart.length === 0 ? (
+          <h1 className="cart-heading">No Items in Cart</h1>
+        ) : (
+          <section className="cart-card-container">
+            <div className="card-product-container">
+              {cart.map((product) => {
+                return (
+                  <>
+                    <div className="cart-card">
```
Comment (t007rushi) [inline src/Pages/Cart.js:122]: provide key prop here

File: src/css/homepage.css [hunk 5]
```diff
@@ -22,25 +24,45 @@
...
+  max-width: 500px;
+  line-height: 3rem;
+  font-size: 1.5rem;
+  font-weight: 800;
+}
+.hero-content-container span {
+  color: #a97559;
+}
+.hero-img-btn {
+  background-color: #a97559;
```
Comment (t007rushi) [inline src/css/homepage.css]: use color variables

File: src/css/product.css [hunk 6]
```diff
@@ -63,15 +65,16 @@
   display: flex;
   max-width: 17rem;
   flex-direction: column;
-  row-gap: 9px;
   border: 1px solid var(--light-gray);
-  padding: 1rem;
+  justify-content: space-evenly;
   position: relative;
+  padding: 0.5rem 0.5rem;
```
Comment (t007rushi) [inline src/css/product.css]: only 0.5 rem works

**Review by t007rushi — APPROVED** [review]
UI is breaking sometimes just look for errors
assign default routes if possible
spacing is very large
Look for suggestions

File: src/Components/Main.js [hunk 7]
```diff
@@ -5,12 +5,21 @@ const Main = () => {
...
+            No matter where you are in the world, you are at <span>home</span>{" "}
+            when <span>tea</span> is served
+          </h1>
+          <Link to="/products">
+            <button className="btn btn-primary hero-img-btn">Buy Now</button>
+          </Link>
+        </div>
+        <img src="../assets/chai.jpg" className="hero-img" />
       </section>
       <section class="hero-card-container">
```
Comment (abhijitdotsharma) [inline src/Components/Main.js]: change to className

File: src/Components/MainCategories.js [hunk 8]
```diff
@@ -4,31 +4,33 @@ import { Link } from "react-router-dom";
...
-            <img src="../assets/exotic.png" className="categories-img" />
-            <h3>Exotic</h3>
+            <img src="../assets/tea-sachet.png" className="categories-img" />
+            <h3>Single Serve</h3>
           </div>
         </Link>
         <Link to="/products">
           <div className="categories">
-            <img src="../assets/flavored.png" className="categories-img" />
+            <img src="../assets/lemon-tea.png" className="categories-img" />
```
Comment (abhijitdotsharma) [inline src/Components/MainCategories.js]: Its good practice to give an alt prop with img tag

**Review by abhijitdotsharma — APPROVED** [review]
Look into what the other review has said, rest looks good to me.

File: src/Components/Main.js (hunk 1 above)
Comment (MarkVed17) [inline src/Components/Main.js]: You shall remove this comment

File: src/Pages/Cart.js [hunk 9]
```diff
@@ -67,9 +75,15 @@ const Cart = () => {
     });
     const convertedJSON = await postData.json();
   };
-  const totalPrice = cart.reduce((acc, cur) => {
-    return (acc = acc + cur.price * cur.qty + 49);
-  }, 0);
+  const totalPrice = () => {
+    console.log(cart, "in func");
```
Comment (MarkVed17) [inline src/Pages/Cart.js]: You may remove this too

File: src/css/homepage.css [hunk 10]
```diff
@@ -1,19 +1,21 @@
...
+  row-gap: 2rem;
   justify-content: space-evenly;
 }
 .categories {
   display: flex;
   flex-direction: column;
   align-items: center;
-  justify-content: center;
+  justify-content: space-between;
+  /* border: 1px solid gray; */
```
Comment (MarkVed17) [inline src/css/homepage.css]: This too

File: src/css/homepage.css [hunk 11]
```diff
@@ -50,26 +72,33 @@
...
-  width: 60%;
+  width: 50%;
   text-align: center;
 }
 .hero-card-content {
   display: flex;
   flex-direction: column;
   align-items: flex-start;
+  row-gap: 0.75rem;
+  /* padding: 1rem 0; */
```
Comment (MarkVed17) [inline src/css/homepage.css]: Here as well

File: src/css/homepage.css [hunk 12]
```diff
@@ -50,26 +72,33 @@
...
 }
 .cta {
   margin-top: 1rem;
 }
+.hero-btn {
+  padding: 0.5rem 1rem;
+}
 @media screen and (max-width: 768px) {
-  .display-categories-container {
+  /* .display-categories-container {
```
Comment (MarkVed17) [inline src/css/homepage.css:101]: All of these

**Review by MarkVed17 — APPROVED** [review]
I liked the naming conventions followed.
Good to proceed!

**Pull request merged**